src/main/java/Event.java

```diff
@@ -0,0 +1,11 @@
+public class Event extends Task{
+    public Event(String input, String start, String end) {
+        super(input, start, end);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString() + " (from: " + this.getStart() +  " to " + this.getEnd() + ")";
```

Comment:
I like how the whitespaces surround the operators, avoiding a common mistake and making the code neat.